### DISCUSSION TITLE
Cannot use ClearFilter on DataGrid - can someone explain it how to implement this?

### DISCUSSION THREAD

**Discussion by MGabala** [Megabit/Blazorise#4671] [Q&A]
Hello i have been looking for this around the net but i cannot find anything. 

Could someone explain and show me how to use this: 

`ClearFilter' on DataGrid:

this is my real example:

```
<DataGrid TItem="OutputClass"
                Data="@employeeList"
                Resizable
                Filterable
                RowSelectable=@((x)=> x.Item.Nazwa != "")
                Responsive
                Bordered
                Narrow="true"
                Striped="true"
                PageSize="350"
                ShowPager
                FixedHeader
                FixedHeaderDataGridHeight="850px"
                FixedHeaderDataGridMaxHeight="1000px"
                PagerPosition="DataGridPagerPosition.Top">

        <DataGridCommandColumn />
        <DataGridColumn Field="@nameof(OutputClass.Percentage)" Caption="% zapasu" Editable />
        <DataGridColumn Field="@nameof(OutputClass.Nazwa)" Width="300px" Caption="Nazwa" Editable />
        <DataGridColumn Field="@nameof(OutputClass.Opis1)" Caption="Opis" Editable />
        <DataGridColumn Field="@nameof(OutputClass.Kod)" Caption="Kod" Editable />
</DataGrid>
```

**Comment by stsrki** [comment]
It is not possible at the moment. We'll see if it can be done https://github.com/Megabit/Blazorise/issues/1403#issuecomment-1465775504

**Comment by MGabala** [comment]
@stsrki is there any other way to remove all filters, all searches in one click?

**Reply by David-Moreira** [reply]
Hello @MGabala 
Can you check our datagrid demo here?
https://bootstrapdemo.blazorise.com/tests/datagrid
Our standard Clear Filter should already clear every filter. Is this what you're looking for?

**Reply by MGabala** [reply]
Yes this will do the job. Btw i am looking for some way to apply multi choice filter? Like this in City column but with availibility to choose more than 1?

**Reply by David-Moreira** [reply]
I believe you can yes.
Look at the **Custom Column Filtering** example here:
https://blazorise.com/docs/extensions/datagrid/features/filtering

It's showcasing a custom Gender filter with a regular select. In theory you can just change it to be a multiple choice component and update the custom filter accordingly.
Let us know if this helps.
